### ISSUE TITLE
HTW should follow the WCAG a11y guidelines

### ISSUE THREAD

**Issue by noa-santo** [Entkenntnis/htw#672]
In germany after June 28, 2025 the Accessibility Strengthening Act (BFSG) will come in effect and requires all websites to be accessible. To make sure that is the case the **WCAG** **_AA_** or at least **_A_** a11y guidelines should be followed (for example a logical tab order).

Because of the small size this site is probably not *required* to follow them but it would still be a good feature.

Links:
- Checklist: [https://www.a11yproject.com/checklist/](https://www.a11yproject.com/checklist/)
- https://weventure.de/en/blog/digital-accessibility-will-become-law-in-2025
- Law: https://www.bgbl.de/xaver/bgbl/start.xav?startbk=Bundesanzeiger_BGBl&jumpTo=bgbl121s2970.pdf#/switch/tocPane?_ts=1747120393193

**Comment by Entkenntnis** [comment]
Have you concrete examples where this could be improved?

**Comment by Entkenntnis** [comment]
Some results:

https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fhack.arrrg.de%2F&flag=us

![Image](https://github.com/user-attachments/assets/3de593bc-4685-4358-acd3-7a0ff258a30d)
